app/main/views/frameworks.py

```diff
@@ -839,13 +840,21 @@ def framework_updates(framework_slug, error_message=None, default_textbox_value=
         file['path'] = '/'.join(path_parts[2:])
         files[path_parts[3]].append(file)
 
+    errors = {CLARIFICATION_QUESTION_NAME: {
+        "text": error_message,
+        "href": f"#{CLARIFICATION_QUESTION_NAME}",
```

Comment:
Normally I prefer f-strings for any sort of string manipulation, but I wonder if in this case it would be more readable to do simple concatenation, as for a second here I thought I was reading Ruby... What are your thoughts?

Reply:
Hmm yes I see your point. I don't think I have a strong opinion either way so I'm happy to change it